doc/source/install.txt

```diff
@@ -1,53 +1,153 @@
-Pre-built installation
+Installing scikit-image
+-----------------------
+If you are on Mac OS X you're lucky, open the terminal and install
```

Comment:
Surely you did not forget that OSX is the world's most advanced operating system?

And that the latest MacBook is also available in gold?